### DISCUSSION TITLE
New option suggestions

### DISCUSSION THREAD

**Discussion by kleenmr** [lidawei1975/colmarvista#49] [Ideas]
1. Brushing options once it is zoomed in!
2. Availability to change the sequence of overlayed spectra. I hope the beneath spectrum is possible to come front by changing the sequence of spectra.

Thank you so much!!

**Comment by lidawei1975** [comment]
Thanks for your comment.  I don't fully understand your comment 1, will work on comment 2.

**Comment by kleenmr** [comment]
Thank you,

Comment 1 is the one I mentioned at the group meeting. Even though the spectrum is zoomed in, I hope I can swipe left or right, up or down to move other chemical shift ranges without zooming out.
